packages/firebase_remote_config/firebase_remote_config/ios/Classes/Reachability.h

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright (C) 2016 Apple Inc. All Rights Reserved.
+ See LICENSE.txt for this sample’s licensing information
+ Abstract:
+ Basic demonstration of how to use the SystemConfiguration Reachablity APIs.
+ */
```

Comment:
Should we change licensing files? Or link to the original file?

Reply:
I'll update the license headers 👍

Reply:
I'm also going to add an additional log as it is worthwhile diagnosing if they have a network connection at all, not just whether remote-config API is reachable.

Reply:
Removed completely now. See https://github.com/firebase/flutterfire/pull/10376#discussion_r1098530818